cfme/tests/openstack/cloud/test_flavors.py

```diff
@@ -0,0 +1,142 @@
+"""Tests for Openstack cloud Flavors"""
+
+import fauxfactory
+import pytest
+
+from cfme.cloud.instance.openstack import OpenStackInstance
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils.appliance.implementations.ui import navigator, navigate_to
+from cfme.utils.version import current_version
+from cfme.utils.wait import wait_for
+from widgetastic.utils import partial_match
+
+pytestmark = [
+    pytest.mark.usefixtures("setup_provider"),
+    pytest.mark.provider([OpenStackProvider], scope='function')
+]
+
+
+DISK_SIZE = 1
+RAM = 64
+VCPUS = 1
+SWAP_SIZE = 1
+RXTX = 1
+ZERO_DISK_SIZE = 0
+
+
+@pytest.fixture(scope='function')
+def flavor(appliance, provider):
```

Comment:
Where is this fixture used? Is it for future test cases?

Reply:
It is not used, so maybe better off be removed